packages/nimbus/src/components/field-errors/field-errors.recipe.tsx

```diff
@@ -0,0 +1,11 @@
+import { defineSlotRecipe } from "@chakra-ui/react";
+
+export const fieldErrorsRecipe = defineSlotRecipe({
+  className: "nimbus-field-errors",
+  slots: ["root"],
+  base: {
+    root: {
+      display: "block",
+    },
+  },
+});
```

Comment:
We could have made this component without slots entirely, but I kept a minimum of boilerplate here in case it needed to be quickly extended now or later